linkis-public-enhancements/linkis-datasource/datasourcemanager/server/src/main/java/com/webank/wedatasphere/linkis/datasourcemanager/core/restful/DataSourceAdminRestfulApi.java

```diff
@@ -63,148 +63,122 @@ public void initRestful(){
         this.formDataTransformer = FormDataTransformerFactory.buildCustom();
     }
 
-    @POST
-    @Path("/env/json")
-    public Response insertJsonEnv(DataSourceEnv dataSourceEnv, @Context HttpServletRequest req){
-        return RestfulApiHelper.doAndResponse(()->{
```

Comment:
About RestfulApiHelper.doAndResponse 
may be modfify  RestfulApiHelper.doAndResponse return  object of Response to Message will be better